prepare-toolchain

```diff
@@ -56,40 +56,17 @@ cd swift-source
 git clone https://github.com/apple/swift.git
 swift/utils/update-checkout --clone --tag "swift-${version}-RELEASE"
 
+ln -s ./llvm-project/llvm ./llvm
+ln -s ./llvm-project/clang ./clang
+
 # patch clang
+info "Patching Clang"
 sed -i '' -e 's;__has_include(<CoreServices/CoreServices.h>);__has_include(<FSEvents/FSEvents.h>);g' clang/lib/DirectoryWatcher/DirectoryWatcher.cpp
 
 # compile swift
+# see here for preset info: https://github.com/apple/swift/blob/master/utils/build-presets.ini#L2105
 info "Compiling Swift"
-swift/utils/build-script \
---build-subdir=LLVMClangSwift_iphoneos \
---release \
--- \
---install-swift \
---install-destdir=output \
---swift-install-components="compiler;clang-builtin-headers;editor-integration;tools;sourcekit-xpc-service;swift-remote-mirror;swift-remote-mirror-headers;" \
---install-prefix=/usr \
---cross-compile-hosts=iphoneos-arm64 \
---compiler-vendor=apple \
---swift-primary-variant-sdk=IOS \
---swift-primary-variant-arch=arm64 \
---build-toolchain-only=1 \
---build-swift-static-stdlib=0 \
---build-swift-dynamic-stdlib=0 \
---build-swift-static-sdk-overlay=0 \
---build-swift-dynamic-sdk-overlay=0 \
---build-runtime-with-host-compiler=0 \
---skip-local-host-install \
---skip-build-benchmarks \
---skip-test-sourcekit \
---skip-test-swift \
---llvm-include-tests=0 \
---llvm-enable-modules=0 \
---build-swift-examples=0 \
---swift-include-tests=0 \
---darwin-deployment-version-ios=10.0 \
---darwin-crash-reporter-client=1 \
---reconfigure
+./swift/utils/build-script --preset "buildbot_iphoneos_arm64_crosscompiler"
```

Comment:
as for 5.2.4 still fails on ninja script unknown target 'swift-test-stdlib-macosx-x86_64'